giga_connectome/atlas.py

```diff
@@ -1,13 +1,15 @@
-import os
-import json
-from typing import Union, List
+from __future__ import annotations
```

Comment:
necessary for python 3.8 if you want to use annotations like 

```
foo : list[str] = ["bar"]
```

instead of doing

```
from typing import List

foo : Lis[str] = ["bar"]
```